docs/articles/documentation/test-api/A-Z.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: docs
+title: A-Z
+permalink: /documentation/test-api/a-z.html
+---
+# A-Z
+
+This topic contains an alphabetical list of all the test API functions and objects with members.
+
+## Functions
```

Comment:
I wonder what's right for this - `global functions` or maybe `factory functions` (like `createTestCafe`)?